server.js

```diff
@@ -9,11 +11,102 @@ app.use(cors());
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
 
-app.get("/", function (request, response) {
-  response.send("Hotel booking server.  Ask for /bookings, etc.");
+//Create a new booking
+app.post("/bookings", (req, res) => {
+  const {
+    title,
+    firstName,
+    surname,
+    email,
+    roomId,
+    checkInDate,
+    checkOutDate,
+  } = req.body;
+
+  //check all field been filled up
+  if (
+    !title ||
+    !firstName ||
+    !surname ||
+    !email ||
+    !roomId ||
+    !checkInDate ||
+    !checkOutDate
+  ) {
+    res.send("newBooking has empty field");
```

Comment:
status

Reply:
you're sending strings with .send and json with .json. always use json 

`res.json({msg: "newBooking has empty field"});`